mono_repo/lib/src/commands/github/action_info.dart

```diff
@@ -35,7 +36,7 @@ enum ActionInfo implements Comparable<ActionInfo> {
   codecov(
     name: 'Upload coverage to codecov.io',
     repo: 'codecov/codecov-action',
-    version: 'main',
+    version: 'd9f34f8cd5cb3b3eb79b3e4b5dae3a16df499a70',
```

Comment:
@kevmoo wdyt about parsing out the versions from our own workflow file - which will be updated by dependabot.

We could do it as a build step even, and just generate a file with all the latest hashes in it. Then whenever dependabot runs on this package we would get a failure in the build validation step and be forced to also re-run a build and land them together.

Reply:
(this could be in a follow up to be clear)

Reply:
this is done